src/viur/core/prototypes/skelmodule.py

```diff
@@ -110,6 +112,46 @@ def baseSkel(self, *args, **kwargs) -> SkeletonInstance:
         """
         return self._resolveSkelCls(*args, **kwargs)()
 
+    def skel(
```

Comment:
Why don't you implement this directly in `baseSkel`? Now you have to customize every call (`baseSkel()` --> `skel()`). If you would implement this in `baseSkel`, all calls could remain the same and you would be more backwards compatible.

With yet another `*skel` method, it only gets more complicated.